social-providers/ios-profile-google/SoomlaiOSProfileGoogle/SoomlaGooglePlus.m

```diff
@@ -50,47 +62,12 @@ -(NSArray *)scopes {
     return _scopes;
 }
 
-#pragma mark Swizzle URL-schemes routing
-
-static IMP originalImplementation = nil;
-static Method originalMethod = nil;
-static Method swizzledMethod = nil;
-
--(BOOL)openURL:(NSURL *)url {
-    //when original method is swizzled, swizzled method will contain original implementation
-    return [[SoomlaProfile getInstance] tryHandleOpenURL:url sourceApplication:[[NSBundle mainBundle] bundleIdentifier] annotation:nil] ||
-            ((BOOL (*)(id, Method, ...))method_invoke)([UIApplication sharedApplication], swizzledMethod, url);
-}
-
-/*
-    swizzling [UIApplication openURL:] method to intercept opening URLs with 'http'-scheme
-    it's working like method overriding (we'll call original method if swizzled method returns NO) but better
-    it's better than overriding because brings some flavour of DI and doesn't oblige user to make changes with his code :)
-    @param forward sets direction of swizzling: if YES - we're replacing original method with our own, else - we're setting original method
- */
--(void)openURLSwizzle:(BOOL)forward {
-    static BOOL methodReplaced = NO;
-    if (!(forward ^ methodReplaced)) //we can swap method implementations only if this condition if true
-        return;
-    if (originalMethod == nil) {
-        originalMethod = class_getInstanceMethod([UIApplication class], @selector(openURL:));
-        swizzledMethod = class_getInstanceMethod([self class], @selector(openURL:));
-    }
-    method_exchangeImplementations(originalMethod, swizzledMethod);
-    methodReplaced = !methodReplaced; //signalize that implementations was swapped
-    LogDebug(TAG, forward ? @"Method openURL of UIApplication was overrided." : @"Method openURL of UIApplication was re-setted to original.");
-
-}
-
```

Comment:
@vedi, yes, I transferred SOOMLA Google Plus from GPPSignIn -> GIDSignIn (GPPSignIn is deprecated and GPGS doesn't work with GPPSignIn) and GIDSignIn opens login view inside app (not in Safari).